src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -118,27 +119,31 @@ public void SynchronizeWithServer_DisconnectedInTheMiddle_ServerInfoIsReusedAndN
         }
 
         [TestMethod]
-        public async Task SynchronizeWithServer_NotInConnectedMode_StoreCleared()
+        public async Task SynchronizeWithServer_NotInConnectedMode_ServerComponentsNotCalledAndStoreCleared()
         {
             var sonarQubeServer = new Mock<ISonarQubeService>();
             var converter = new Mock<ITaintIssueToIssueVisualizationConverter>();
             var taintStore = new Mock<ITaintStore>();
+            var serverBranchProvider = new Mock<IServerBranchProvider>();
             var logger = new TestLogger();
 
             var testSubject = CreateTestSubject(
                 taintStore: taintStore.Object,
                 sonarService: sonarQubeServer.Object,
                 converter: converter.Object,
                 mode: SonarLintMode.Standalone,
+                serverBranchProvider: serverBranchProvider.Object,
                 logger: logger);
 
             await testSubject.SynchronizeWithServer();
 
             CheckStoreIsCleared(taintStore);
 
+            // Server components should not be called
             logger.AssertPartialOutputStringExists("not in connected mode");
-            sonarQubeServer.Invocations.Count.Should().Be(0);
```

Comment:
Cosmetic: switched existing checks to use `HaveCount`, as this is what is suggested by the Fluent Assertions analyzers.